launch/image_processing.launch

```diff
@@ -0,0 +1,15 @@
+<launch>
+  <arg name="view" default="false" /> 
```

Comment:
Looks like this is unused. You should have a image_view node with a toggle to use this. Or you can remove it